torch_xla/csrc/BUILD

```diff
@@ -146,7 +146,6 @@ ptxla_cc_library(
         "@org_tensorflow//tensorflow/compiler/xla/client/lib:sorting",
         "@org_tensorflow//tensorflow/compiler/xla/client/lib:svd",
         "@org_tensorflow//tensorflow/compiler/xla/stream_executor:dnn",
-        "@org_tensorflow//tensorflow/core:lib",
```

Comment:
lol I am surprised that this is just a dead dependency..

Reply:
me too!

and according to https://github.com/pytorch/xla/pull/5174/files, `torch_xla/csrc/runtime:util` is the only way to include all the deps from `core:lib`